packages/block-renderer/src/components/pattern-renderer.tsx

```diff
@@ -20,13 +20,22 @@ const PatternRenderer = ( {
 	viewportHeight,
 	minHeight,
 	maxHeight,
-	prependHtml = '',
+	transformHtml,
 	shouldShufflePosts,
 }: Props ) => {
 	const renderedPatterns = usePatternsRendererContext();
 	const pattern = renderedPatterns[ patternId ];
-	const patternHtml = usePatternMinHeightVh( prependHtml + pattern?.html, viewportHeight );
-	const inlineCss = usePatternInlineCss( patternId, patternHtml, shouldShufflePosts );
+
+	let patternHtml = pattern?.html;
+	if ( viewportHeight ) {
+		patternHtml = normalizeMinHeight( patternHtml, viewportHeight );
```

Comment:
As we remove the early check, we have to ensure the `patternHtml` is string before normalizing or transforming

Reply:
Ooops, you're right, fixed, thanks! (But somehow it worked locally)

Reply:
It works if the cache is not expired 😆